addons/payment/static/src/js/post_processing.js

```diff
@@ -89,6 +89,14 @@ publicWidget.registry.PaymentPostProcessing = publicWidget.Widget.extend({
         this.call('ui', 'unblock');
         const statusContainer = document.querySelector('div[name="o_payment_status_content"]');
         statusContainer.innerHTML = renderToElement(xmlid, display_values).innerHTML;
+        if (display_values.state === 'pending') {
+            const contactUsLink = statusContainer.querySelector('a[href="#discussion"]');
+            if (contactUsLink) {
+                let currentHref = contactUsLink.getAttribute('href');
+                currentHref = `${display_values.landing_route}${currentHref}`;
+                contactUsLink.setAttribute('href', currentHref);
+            }
+        }
```

Comment:
Seems useless to me. Just pass the contactus page in the message. And, in website_sale, change it with the `contact_us_button_url` instead. =] This can be done in python directly

Reply:
> Seems useless to me. Just pass the contactus page in the message.

same comment as below